ddp-workspace/projects/ddp-sdk/src/lib/services/analyticsEvents.service.ts

```diff
@@ -23,6 +28,12 @@ export class AnalyticsEventsService {
       eventValue
     };
     this.events.next(event);
+    this.events.next(this.buildGTagEvent(eventCategory, eventAction));
+  }
+
+  public emitCustomGtagEvent(eventName: string, clickText?: string, clickUrl?: string) {
+    const newEvent = this.buildGTagEvent(eventName, clickText, clickUrl);
+    this.events.next(newEvent);
```

Comment:
If we know we only care about GtagEvents, then we create only those specifically. This is what we are using from Singular static site.